stress/v2/ponyExpress/ponyExpress_write_test.go

```diff
@@ -0,0 +1,84 @@
+package ponyExpress
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
```

Comment:
Probably not. Was going to try and write some handler functions for the test server to mimic an influx server a little better. Those two files (`stress/v2/ponyExpress/ponyExpress_write_test.go`,`stress/v2/ponyExpress/ponyExpress_query_test.go`) have the weakest testing